desktop/app/console-helper.js

```diff
@@ -1,9 +1,9 @@
 import {ipcMain, ipcRenderer} from 'electron'
 import util from 'util'
-import getenv from 'getenv'
+import {forwardLogs} from '../../react-native/react/local-debug'
```

Comment:
Using a specific flag to control logging so we can have devtools but maybe no logging